### PR TITLE
Improve Minnano-AV scraper

### PR DESCRIPTION
**Pull request by Clasp8775**
- Automatically convert sizes from Japanese to US
- Properly add and de-duplicate aliases
- Set correct name order for StashDB (Last First -> First Last)
- Search for both name orders to simplify the process

1. I'm not very proficient in Python so feel free to suggest changes.
3. Some of the functions (especially converting bra sizes) might be better suited for py_common?

### CODE REVIEW

**Review by Maista6969 — CHANGES_REQUESTED** [review]
A great improvement over the XPath scraper! Happy to merge this once it grabs all aliases 👍

File: scrapers/Minnano-AV/Minnano-AV.py [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+    "name_kanji": '//section[@class="main-column details"]/h1/text()',
+    "origin": "//span[text()='出身地']/../p/a/text()",
+    "name": '//section[@class="main-column details"]/h1/span/text()',
+    "search_url": '../h2[@class="ttl"]/a/@href',
+    "search": '//p[@class="furi"]',
+    "twitter": ("//span[text()='ブログ']/../p/a[contains(@href,'twitter.com')]/@href"),
+}
+
+REGEXES = {
+    "alias": r"(?P<kanji>[^\x29\uFF09]+?)(?P<studio>[\x28\uFF08\u3010][^\x29\uFF09\u3011]+(?:[\x29\uFF09\u3011]))? [\x28\uFF08](?P<katakana>\w+)? / (?P<romanized>[a-z-A-Z ]+)?[\x29\uFF09]",
```
[the remark this answers is not in the document]
Comment (Clasp8775) [reply scrapers/Minnano-AV/Minnano-AV.py]: Added your fix

File: scrapers/Minnano-AV/Minnano-AV.py [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+    else:
+        log.error("No argument processed")
+        log.debug(stdin)
+
+
+if __name__ == "__main__":
+    try:
+        main()
+    except Exception as e:
+        logging.exception(e)
```
[the remark this answers is not in the document]
Comment (Clasp8775) [reply scrapers/Minnano-AV/Minnano-AV.py]: Done

**Comment by Clasp8775** [comment]
I've tested the alias regex on a few variations but that's the first time seeing that one! I've added a link to regex101 with the test strings, also putting it here for visibility: https://regex101.com/r/9k2GXw/5

**Pull request merged**